### PR TITLE
Add com.byteseb.Duel

### PR DESCRIPTION
**Pull request by ByteSeb**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.byteseb.Duel.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "com.byteseb.Duel",
```
Comment (hfiguiere) [inline com.byteseb.Duel.json:2]: `byteseb.com` doesn't seem to exist, so the app-id doesn't meet the requirements.

File: com.byteseb.Duel.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "Duel",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/ByteSeb/Duel/archive/refs/tags/v0.1.0.tar.gz"
```
Comment (hfiguiere) [inline com.byteseb.Duel.json:32]: this is not a git source.

File: com.byteseb.Duel.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "Duel",
+            "builddir" : true,
```
Comment (hfiguiere) [inline com.byteseb.Duel.json:27]:
```suggestion
```

it's unecessary.

**Comment by ByteSeb** [comment]
Sorry. Updated the manifest and fixed the mistakes you pointed out

File: com.byteseb.Duel.json (hunk 2 above)
Comment (hfiguiere) [reply com.byteseb.Duel.json:31]: and checksum is required.

**Comment by hfiguiere** [comment]
not all.

File: com.byteseb.Duel.json (hunk 1 above)
Comment (hfiguiere) [reply com.byteseb.Duel.json:2]: Until you fix this, this PR can't move forward.

File: com.byteseb.Duel.json (hunk 2 above)
Comment (hfiguiere) [reply com.byteseb.Duel.json:31]: Until you fix this, this PR can't move forward.

**Pull request closed without merging**